### PR TITLE
Add moon.

### PR DESCRIPTION
**Pull request by milesj**
Fixes https://github.com/nrwl/monorepo.tools/issues/92

Screenshot 2023-06-14 at 2 30 02 PM
Screenshot 2023-06-14 at 2 30 14 PM

### CODE REVIEW

File: libs/website/ui-home/src/lib/tools-review.tsx [hunk 1]
```diff
@@ -180,8 +193,9 @@ const understandable: Item[] = [
       { title: 'Bazel', value: 'manualImplementation' },
       { title: 'Gradle', value: 'supported' },
       { title: 'Lage', value: 'supported' },
-      { title: 'Nx', value: 'supported' },
```
Comment (milesj) [inline libs/website/ui-home/src/lib/tools-review.tsx:183]: Order was wrong here, not sure if it mattered.

**Comment by milesj** [comment]
@bcabanes Would appreciate a review when you have a chance!

**Comment by milesj** [comment]
Bump.

**Review by ZackDeRose — APPROVED** [review]
LGTM

**Pull request merged**